tests/controllers_tests/controllers_test.go

```diff
@@ -107,8 +108,8 @@ func (h dummyConceptDataModel) RetriveAllBySourceId(sourceId int) ([]*models.Con
 func (h dummyConceptDataModel) RetrieveInfoBySourceIdAndConceptIds(sourceId int, conceptIds []int) ([]*models.ConceptSimple, error) {
 	// dummy data with _some_ of the relevant fields:
 	conceptSimple := []*models.ConceptSimple{
-		{ConceptId: 1234, ConceptName: "Concept A"},
-		{ConceptId: 5678, ConceptName: "Concept B"},
+		{ConceptId: 1234, ConceptName: "value1"},
+		{ConceptId: 5678, ConceptName: "value2"},
```

Comment:
is this change needed?

Reply:
but `RetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIds` is not supposed to return concept names, but concept values.